tests/datatypes/test_complex.py

```diff
@@ -39,6 +39,16 @@ def test_equality_with_numbers_when_zero_imag(self):
             print(y.__ne__(x))
             """)
 
+    def test_hash(self):
+        self.assertCodeExecution("""
+            print(hash(complex(2, 5)))
+            print(hash(complex(2, 0)))
+            print(hash(complex(20893129038, 1903813213)))
+            print(hash(complex(12908.19238, 1293.19238)))
+            print(hash(complex(0, 0)))
+        """)
+
+
 
 class UnaryComplexOperationTests(UnaryOperationTestCase, TranspileTestCase):
```

Comment:
At column 1: (E303) too many blank lines (3)

Reply:
Thanks so much Brutus-the-Bee, I fixed it.